module/behaviour/strategy/PS3Walk/src/Joystick.hpp

```diff
@@ -12,62 +12,62 @@
 // along with this program.  If not, see <http://www.gnu.org/licenses/>
 
 #ifndef __JOYSTICK_HH__
-#define __JOYSTICK_HH__
+    #define JOYSTICK_HH_
 
-#include <string>
+    #include <string>
 
-#define JS_EVENT_BUTTON 0x01  // button pressed/released
-#define JS_EVENT_AXIS   0x02  // joystick moved
-#define JS_EVENT_INIT   0x80  // initial state of device
+    #define JS_EVENT_BUTTON 0x01  // button pressed/released
+    #define JS_EVENT_AXIS   0x02  // joystick moved
+    #define JS_EVENT_INIT   0x80  // initial state of device
```

Comment:
```suggestion
enum struct JS_EVENT : unsigned char {
    BUTTON = 0x01, AXIS = 0x02, INIT = 0x80
};
```
Probably a more c++ way of doing things. Also put in the namespace.

Reply:
probs another pr

Reply:
Agreed, there is a more c++ way of doing this that should probably be used